storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -3182,6 +3360,11 @@ static void data_copy_thread_func(data_thread_ctxt_t *ctxt) {
     if (xtrabackup_copy_datafile(node, num)) {
       xb::error() << "failed to copy datafile " << node->name;
       *(ctxt->error) = true;
+    } else {
+      if (ddl_tracker && fsp_is_undo_tablespace(node->space->id)) {
+        ddl_tracker->add_undo_tablespace(node->space->id,
+                                         node->space->files.front().name);
+      }
```

Comment:
Add a comment that, instead of tracking at the beginning, we will do the tracking here after xtrabackup copies of the undo files.